### DISCUSSION TITLE
New Roms which stored on SD not show up in Onion UI

### DISCUSSION THREAD

**Discussion by P4tr1zzle** [OnionUI/Onion#1234] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.2.2

### Miyoo Firmware Version

202305052130

### Provide a clear and concise description of the issue

I updated from Onion OS 4.2 RC to 4.2.2, stored a few new Games (.gba) in the GBA-folder, scrapped the Artwork (which seems to work fine), refreshed the GBA-List but no game show up. When i open them in Retroarch, they are working fine. Am i doing s.t. wrong or ist it a Bug. In 4.2 RC, i have no issues. 

### Steps to Reproduce

-"-

### Provide any additional context or information that may help us investigate your issue

-"-

**Comment by Allismeaningless** [comment]
I'm not technical but have you tried manually deleting the game list cache on the SD card then doing another refresh. Also, is the filetype one that's supported?

**Reply by P4tr1zzle** [reply]
Thanx for your reply. I have deleted the cache-list, but its not working.

The games are working fine if i load them in Retroarch , so the filetype should not be the the problem.

**Reply by Phanturian** [reply]
Is there a miyoogamelist.xml present in your Roms/GBA folder? If so, back it up on your computer and try deleting the miyoogamelist.xml in the GBA folder on your sd card. I haven't used scraper on the Miyoo Mini Plus for quite a while, so I'm not sure if it generates a miyoogamelist.xml (it used to just scrape covers). If a miyoogamelist.xml exists, open it with a text editor and see if it's correctly pointing to your roms. If not, none of your games will show even though they are in the GBA folder.

**Reply by P4tr1zzle** [reply]
Hi there,

the xml-file is in the GBA folder. I have deleted the file and now it works. Thank you. But it is a bit strange, that i have to delete the file to refresh the game list.

**Reply by Phanturian** [reply]
I'm pretty sure the miyoogamelist.xml overrides everything else when it's present. So if you have the xml present, but none of your games are listed correctly within the text of the xml, they just won't show. Glad it works now, hopefully everything is presented the way you wanted.
